README.md

```diff
@@ -10,8 +10,9 @@
   - iOS, 
   - NodeJS
   - Python (looking for a maintainer)  
+- Overview of [implemented Aries protocols](./aries.md)
 - Aries Framework Rust requires [mediator agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md).
-  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).  
+  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).    
   
 # Work in progress
 - This is spin off what has previously been known as LibVCX library. 
```

Comment:
We are not a spinoff :) We are the continuation of it, an evolution :)

Reply:
Until `IndySDK libvcx` is not completely abolished, I think it's accurate to say we are spinoff / fork. I saw Evernym still recently comitting to `IndySDK libvcx`, and some users are only now migrating from `IndySDK libvcx` to `aries-vcx`

Reply:
OK